src/hue/api/room.rs

```diff
@@ -6,6 +6,7 @@ use crate::hue::api::{RType, ResourceLink};
 pub struct RoomMetadata {
     pub name: String,
     pub archetype: RoomArchetype,
+    pub hidden: bool,
```

Comment:
Whoah, not so fast 😏

This is the hue *api model*. Hue has no concept of a `.hidden` field on RoomMetadata, so this is wrong.

(everything in `src/hue` is pretty much code to interface to/from hue-specific json blobs)